{"variants":[{"paths":["\/documentation\/niohttp2\/http2errorcode\/connecterror"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/connectError","interfaceLanguage":"swift"},"abstract":[{"text":"The connection established in response to a ","type":"text"},{"code":"CONNECT","type":"codeVoice"},{"text":" request","type":"text"},{"text":" ","type":"text"},{"text":"was reset or abnormally closed.","type":"text"}],"schemaVersion":{"patch":0,"minor":3,"major":0},"sections":[],"primaryContentSections":[{"declarations":[{"platforms":["Linux"],"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","kind":"typeIdentifier"}],"languages":["swift"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode"]]},"metadata":{"externalID":"s:8NIOHTTP214HTTP2ErrorCodeV07connectC0ACvpZ","roleHeading":"Type Property","symbolKind":"property","role":"symbol","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","kind":"typeIdentifier"}],"title":"connectError","modules":[{"name":"NIOHTTP2"}]},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode":{"fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"HTTP2ErrorCode"}],"type":"topic","abstract":[{"type":"text","text":"An HTTP\/2 error code."}],"role":"symbol","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","title":"HTTP2ErrorCode","kind":"symbol","navigatorTitle":[{"kind":"identifier","text":"HTTP2ErrorCode"}],"url":"\/documentation\/niohttp2\/http2errorcode"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","title":"NIOHTTP2","kind":"symbol","role":"collection","url":"\/documentation\/niohttp2","type":"topic","abstract":[]},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode/connectError":{"type":"topic","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/connectError","url":"\/documentation\/niohttp2\/http2errorcode\/connecterror","abstract":[{"text":"The connection established in response to a ","type":"text"},{"type":"codeVoice","code":"CONNECT"},{"text":" request","type":"text"},{"text":" ","type":"text"},{"text":"was reset or abnormally closed.","type":"text"}],"kind":"symbol","title":"connectError","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","kind":"typeIdentifier"}],"role":"symbol"}}}